tests/e2e/specs/NavTest.spec.js

```diff
@@ -0,0 +1,20 @@
+// https://docs.cypress.io/api/introduction/api.html
+
+describe("Test navigation", () => {
+  it("Navigation should exist", () => {
+    cy.visit("/");
+    cy.get("nav").should("exist");
+  });
+  it("Navigation should be clickable", () => {
+    cy.get("[data-cy='nav-elements']").click({ multiple: true });
+  });
+  it("Navigation elements should be focusable", () => {
+    // cypress can't test focus on these elements
```

Comment:
You leared how =)

Reply:
can't test focus because cypress doesn't see list elements as focusable elements. it only works on input fields.